### PR TITLE
Document how to solve physics vector equations

### PR DESCRIPTION
**Pull request by tjstienstra**
#### References to other Issues or PRs

Replaces #24923

#### Brief description of what is fixed or changed
Document how to solve equations in the form of vectors in `sympy.physics.vector`.

#### Release Notes

NO ENTRY

### CODE REVIEW

File: doc/src/modules/physics/vector/advanced.rst [hunk 1]
```diff
@@ -198,3 +198,34 @@ so dynamic symbols created before or after will print the same way.
 Also note that ``Vector``'s ``.dt`` method uses the ``._t`` attribute of
 ``dynamicsymbols``, along with a number of other important functions and
 methods. Don't mix and match symbols representing time.
+
+Solving Vector Equations
+========================
+
+To solve equations involving vectors, you cannot directly use the solve
+functions on a vector. Instead, you must convert the vector equation to a
+matrix.
```
Comment (moorepants) [inline doc/src/modules/physics/vector/advanced.rst:207]: to a set of scalar equations

the fact that you make it a matrix is just an implementation choice

**Comment by moorepants** [comment]
LGTM

One comment above for a possible slight improvement.

File: doc/src/modules/physics/vector/advanced.rst (hunk 1 above)
Comment (tjstienstra) [reply doc/src/modules/physics/vector/advanced.rst]: Addressed in cbc63c2aecb6b2567ebcf1fc36a2d72a06c815bc

**Comment by moorepants** [comment]
Thanks!

**Pull request merged**